### DISCUSSION TITLE
Hubs admin run locally, can not access server API

### DISCUSSION THREAD

**Discussion by baohiep83** [mozilla/hubs#4631] [General Questions & Discussion]
I run hubs admin locally, when I login hubs admin page, I get errors as below picture. When I click on any function of menu, system logout automatically and show the infor message "Your session has ended, please reconnect". Please help me to fix this issue.
![image](https://user-images.githubusercontent.com/42344684/132978882-446d01f8-2a01-423f-88cf-c06994f01a5c.png)

**Comment by pattersonbl2** [comment]
are you trying to do any custom work ? 

What were your steps to deploying ?   

From the looks of the, you're having a connection issue with ita which is a locked-down service for hubs cloud.

**Comment by baohiep83** [comment]
@Pattersonbl2 I installed all source codes of hubs in my computer, I want to run all services in my computer so that I can customize some functions. Do you have any solution for this issue, please help me.

**Reply by pattersonbl2** [reply]
I don't have a solution at the moment. It's does look like your local install isn't communicating￼￼ with a service in the server stack. Normally you can use journalctl to view logs on the reticulum service. ￼￼

**Reply by niitbeo** [reply]
you solved the problem without running the localhost server installation

**Comment by pattersonbl2** [comment]
Were you able to make any progress on this issue ?

**Reply by baohiep83** [reply]
@Pattersonbl2 I think when running hubs admin locally, it is not related to api of cloud. I still debug this issue. When I find out the solution, I will update in this page.

**Reply by shakya9650** [reply]
hello  bro can you tell me how to get admin page  which line of code you are modify

**Comment by fortuneCookieGO** [comment]
Hello, may I ask where the interface of the administrator is started? I have run the admin project under HUBS, but it is not the expected management background interface.
